include/trace_logger.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 #include <string>
 
+typedef void(*trace_callback)(int, const char*);
```

Comment:
Where is this used?  Should it be in reinforcement_learning namespace instead of global?

Reply:
It is used by binding_trace.h.  will move it to rl.net.native.